### DISCUSSION TITLE
500 Internal Server Error

### DISCUSSION THREAD

**Discussion by fadejevs** [vercel/community#6664] [Help]
### Summary

Keep getting "500 Internal Server Error." Even though all my env's are correct, helpp

### Steps to Reproduce

When clicking on "Sign In" using Kinde as a provider

**Comment by versecafe** [comment]
@fadejevs Can you provide some error logs to help better investigate the issue?

**Reply by fadejevs** [reply]
yep! 
showing this, even though all of the .env routes are correct, working in localhost too. 
![image](https://github.com/vercel/community/assets/73697657/5a71d5cb-f6a0-4ba6-bb22-d88214e37746)
